### DISCUSSION TITLE
mp4 clips won't play on multiple macs

### DISCUSSION THREAD

**Discussion by raspman** [blakeblackshear/frigate#10731] [General]
When exporting (or downloading) recorded video, it saves as an MP4 file.  I've tried to play on multiple Mac computers with Quicktime and other MP4 players and they all fail.  The only MP4 files that play fine are event clips.  Any adjustment that is needed?  This is the Frigate add-on within HA.

**Comment by NickM-27** [comment]
more info would be needed like error or something. My mac plays all of my clips fine so information about the cameras, metadata read from ffprobe on the ones that work and don't work would also be helpful

**Reply by raspman** [reply]
For whatever reason, newer recorded video is now playing back fine on my Mac, so not sure if there was some previous video that was corrupted, etc..but it does now work.  I will close and thank you.
